src/swap/defi/lifi.ts

```diff
@@ -312,7 +312,7 @@ export function makeLifiPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
         spendTargets: [
           {
             memo: approvalData,
-            nativeAmount: '0',
+            nativeAmount: mul(transactionRequest.value, '1'),
```

Comment:
Yikes. This is the approval transaction and should never send actual funds. This could cause a loss of funds for the user.